app/static/js/volunteerProfile.js

```diff
@@ -0,0 +1,65 @@
+$(document).ready(function(){
+
+  $(".form-check-input").click(function updateInterest(){
+    var programID = $(this).attr('id');
+    var interest = $(this).is(':checked');
+    var username = $(this).attr('name');
+
+    var routeUrl = interest ? "addInterest" : "removeInterest";
+    interestUrl = "/" + username + "/" + routeUrl + "/" + programID ;
+    $.ajax({
+      method: "POST",
+      url: interestUrl,
+      success: function(response) {
+          msgFlash("Your interest has been updated", "success");
+          location.reload(true);  //  Reloading page after user clicks on the show interest checkbox
+      },
+      error: function(request, status, error) {
+        console.log(status,error);
+        msgFlash("Error Updating Interest", "danger");
+        location.reload(true);
+      }
+    });
+  });
+
+  $(".ban").click(function() {
+
+    $("#banButton").text($(this).val() + " Volunteer");
+    $(".modal-title").text($(this).val() + " Volunteer");
+    $("#modalProgramName").text("Program: " + $(this).attr("name"));
+    $("#banModal").modal("toggle");
+    $("#banButton").attr("programID", $(this).attr("id"))
+    $("#banButton").attr("username", $(".form-check-input").attr("name"))
+    $("#banButton").attr("banOrUnban", $(this).val());
+    $("#ubanEndDate").show()
+    $("#banNoteDiv").hide()
+    $("#banEndDate").val("")
+    $("#banVolunteerNote").val("")
+
+    if( $(this).val()=="Unban"){
+      $("#ubanEndDate").hide()
+      $("#banEndDate").val("0001-01-01") //This is a placeholder value for the if statement in line 49 to work properly
+      $("#banNoteDiv").show()
+      $("#banNote").text($(this).attr("note"))
+    }
+
```

Comment:
Rather than call `$()` multiple times for each element, you should store them each in a variable. That way there aren't as many places you need to change the id should the id change, and you aren't having to make jquery do as much work.